src/SlackEventHandler.ts

```diff
@@ -217,6 +222,7 @@ export class SlackEventHandler extends BaseSlackHandler {
     protected async handleMessageEvent(event: ISlackMessageEvent, teamId: string) {
         const room = this.main.rooms.getBySlackChannelId(event.channel) as BridgedRoom;
         const team = await this.main.datastore.getTeam(teamId);
+        const userOrBotId = event.user || event.bot_id;
```

Comment:
Hmm, careful here. A bot has both a user and bot_id, which are two different things. Why do we want to pass the bot_id here?